script.js

```diff
@@ -0,0 +1,364 @@
+initializeCalculator();
+function initializeCalculator() {
+  let first = {value: "", unsaved: true, saved: false, integer: true};
+  let second = {value: "", unsaved: true, saved: false, integer: true};
+  let operator = {value: "", unsaved: true, saved: false};
+  let result = "";
+  let allowNewNumber = false;
+
+  listenClicks();
+  listenKeys();
+
+  function listenKeys() {
+    window.addEventListener("keydown", event => {
+      const key = document.querySelector(`[data-key="${event.key}"]`);
+      if (key) {
+        bindEvent(key.id);
+        key.focus(); // Safari and Firefox don't handle focus in CSS files
+      }
+    });
+  }
+
+  function listenClicks() {
```

Comment:
`listenToClicks` makes more sense in English

Reply:
I think this function name does not reflect what you are doing in the function. I'd call it `addOnclickEventListenersToButtons` or something like that